scraper.rb

```diff
@@ -5,6 +5,9 @@
 require 'nokogiri'
 require 'open-uri'
 require 'colorize'
```

Comment:
As you've removed the only place that actually _uses_ `colorize`, the require, and the dependency, can be dropped. I've cherry-picked this over to https://github.com/everypolitician-scrapers/morocco-house-of-representatives/pull/6